tools/ovc/openvino/tools/ovc/moc_frontend/pipeline.py

```diff
@@ -90,156 +92,205 @@ def check_places_are_same(places_original: List[Place], places_new: List[Place])
             [item for item in places_original if any(
                 [item.is_equal(item2['node']) for item2 in places_new])]) == len(places_original)
 
-    def add_names_to_tensors(model: InputModel, places: List[Place]):
```

Comment:
this function is moved. please leave it here

Reply:
It is not moved. diff shows it wrong. Please use "hide whitespace" changes feature

Reply:
I don't want to use hide whitespaces. This change is included to all changes

Reply:
I see no value of this movement from one place to another. Moreover, else branch gets longer by this change